mne/preprocessing/ica.py

```diff
@@ -1408,6 +1409,11 @@ def plot_components(self, picks=None, ch_type=None, res=64, layout=None,
             the head circle. If dict, can have entries 'center' (tuple) and
             'scale' (tuple) for what the center and scale of the head should be
             relative to the electrode locations.
+        data : Raw | Epochs | None
```

Comment:
can you call it inst ? we tend to use this for param name for data instance.

data suggests it's an ndarray